### PR TITLE
Feature/userservice

### PR DESCRIPTION
**Pull request by akrojas07**
microservice for user management

### CODE REVIEW

File: UserManagement/User.Infrastructure/Repository/UserRepositories/UserRepository.cs [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+using System.Text;
+using System.Threading.Tasks;
+using User.Infrastructure.Repository.Interfaces;
+using Microsoft.EntityFrameworkCore;
+
+using DbUser = User.Infrastructure.Repository.Entities.UserAccount;
+using User.Infrastructure.Repository.Entities;
+
+namespace User.Infrastructure.Repository.UserRepositories
+{ //Package Manager Console => Scaffold-DbContext "Server=DESKTOP-LQ9NL1I;Database=BudgetApp;Trusted_Connection=True;" Microsoft.EntityFrameworkCore.SqlServer -OutputDir Repository/Entities -f
```
Comment (jruan1580) [inline UserManagement/User.Infrastructure/Repository/UserRepositories/UserRepository.cs:12]: remove this comment

File: UserManagement/User.Infrastructure/Repository/UserRepositories/UserRepository.cs [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+    public class UserRepository : IUserRepository
+    {
+        public async Task CreateNewUserAccount(DbUser user)
```
Comment (jruan1580) [inline UserManagement/User.Infrastructure/Repository/UserRepositories/UserRepository.cs:15]: add method comment

**Review by jruan1580 — CHANGES_REQUESTED** [review]
address comments

**Comment by akrojas07** [comment]
addressed comments

**Review by jruan1580 — APPROVED** [review]

**Pull request merged**